packages/ejson/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 1.0.0
```

Comment:
Should each ejson package have an independent changelog or should we bundle all changes in the same changelog?